### ISSUE TITLE
--limit 10 parameter causes sql operations command to crash

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#449]
```
Issue running command [gcloud sql operations list].

What steps will reproduce the problem?

I tried the following command described in the documenation here: 
https://cloud.google.com/sql/docs/import-export#check

gcloud sql operations --instance YOUR_INSTANCE_NAME list --limit 10

What is the expected output? What do you see instead?

A list of operations limited to the last 10. It crashes instead.

Please provide any additional information below.

I'm using a 2nd gen SQL instance

Traceback (most recent call last):
  File "calliope/cli.py", line 609, in Execute
    result = args.cmd_func(cli=self, args=args)
  File "calliope/backend.py", line 1229, in Run
    display.Displayer(command_instance, args, resources).Display()
  File "calliope/display.py", line 356, in Display
    self._command.Display(self._args, self._resources)
  File "surface/sql/operations/list.py", line 71, in Display
    list_printer.PrintResourceList('sql.operations', result)
  File "core/list_printer.py", line 17, in PrintResourceList
    console_io.PrintExtendedList(items, COLLECTION_COLUMNS[collection])
  File "core/console/console_io.py", line 573, in PrintExtendedList
    for item in items:
  File "core/util/peek_iterable.py", line 137, in next
    item = self._NextItem()
  File "core/util/peek_iterable.py", line 110, in _NextItem
    return self._iterable.next()
  File "third_party/apitools/base/py/list_pager.py", line 46, in YieldFromList
    response = getattr(service, method)(request, global_params=global_params)
  File "third_party/apis/sqladmin/v1beta3/sqladmin_v1beta3_client.py", line 568, in List
    config, request, global_params=global_params)
  File "third_party/apitools/base/py/base_api.py", line 631, in _RunMethod
    method_config, request, global_params, upload, upload_config, download)
  File "third_party/apitools/base/py/base_api.py", line 603, in PrepareHttpRequest
[output truncated]

```

Original issue reported on code.google.com by `jeffv...@gmail.com` on 29 Dec 2015 at 9:21

**Comment by GoogleCodeExporter** [comment]
```
We can reproduce this and will fix it soon--thank you for reporting.
```

Original comment by `z...@google.com` on 4 Jan 2016 at 2:54
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
There should be a fix in the Cloud SDK 92.0.0 release.
```

Original comment by `z...@google.com` on 4 Jan 2016 at 6:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
The Cloud SDK 92.0.0 release is out. Please re-open if you're still 
experiencing this issue.
```

Original comment by `z...@google.com` on 18 Feb 2016 at 8:16
- Changed state: **Fixed**
